src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/MovieRatings
+spring.datasource.username=root
+spring.datasource.password=
```

Comment:
Either smart to not upload your password to a public forum or not smart to not have a password. (Doesn't really matter in this case as its a local db)